docs/zh/navigation.yml

```diff
@@ -436,6 +436,8 @@ nav:
           - 规划:
               - 网卡规划: network/plans/ethplan.md
               - 网络规划: network/plans/netplan.md
+          - 性能报告
+              - 网络 CNI 性能测试报告: network/performance/CNI-performance.md
```

Comment:
pls use small letters, according to our file naming conventions https://github.com/DaoCloud/DaoCloud-docs#naming-conventions

Reply:
> pls use small letters, according to our file naming conventions https://github.com/DaoCloud/DaoCloud-docs#naming-conventions

fixed

Reply:
You need also to change the file name. Tips: only changing upper cases to small ones does not work here; you should also make additional minor changes to make it easier for computer programms to check the difference.

Reply:
> You need also to change the file name. Tips: only changing upper cases to small ones does not work here; you should also make additional minor changes to make it easier for computer programms to check the difference.

thanks for your reminder!